### ISSUE TITLE
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by realnsleo** [ajay2507/ebay-node-api#4]
Awesome module. However, I am getting an error when I try using the searchItems function:

```
ebay.searchItems({
        keyword: 'iphone',
        limit: 3,
        filter: {priceCurrency: 'USD', conditions: 'NEW'},
});
```
The api reports an error in `ebay-node-api\src\buy-api.js:52:34`. I see the issue has something to do with `searchConfig.fieldgroups.length` variable not being seen. Am I missing something here?

Thanks!

**Comment by ajay2507** [comment]
@realnsleo Thanks for pointing out this. Let me dig into this.

**Comment by ajay2507** [comment]
Fixed in version 2.2.1 https://github.com/ajay2507/ebay-node-api/releases/tag/v2.2.1
